2018-11-07T01:30:34.000+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/usr/data/db 64-bit host=2978744f7281
2018-11-07T01:30:34.000+0000 I CONTROL  [initandlisten] db version v4.0.3
2018-11-07T01:30:34.001+0000 I CONTROL  [initandlisten] git version: 7ea530946fa7880364d88c8d8b6026bbc9ffa48c
2018-11-07T01:30:34.001+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2018-11-07T01:30:34.001+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2018-11-07T01:30:34.002+0000 I CONTROL  [initandlisten] modules: none
2018-11-07T01:30:34.002+0000 I CONTROL  [initandlisten] build environment:
2018-11-07T01:30:34.002+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2018-11-07T01:30:34.003+0000 I CONTROL  [initandlisten]     distarch: x86_64
2018-11-07T01:30:34.003+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2018-11-07T01:30:34.003+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true }, storage: { dbPath: "/usr/data/db", mmapv1: { smallFiles: true } }, systemLog: { destination: "file", path: "/usr/data/logs/log.txt" } }
2018-11-07T01:30:34.006+0000 W STORAGE  [initandlisten] Detected unclean shutdown - /usr/data/db/mongod.lock is not empty.
2018-11-07T01:30:34.009+0000 I STORAGE  [initandlisten] Detected data files in /usr/data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2018-11-07T01:30:34.012+0000 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2018-11-07T01:30:34.014+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=487M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2018-11-07T01:30:41.033+0000 I STORAGE  [initandlisten] WiredTiger message [1541554241:33049][1:0x7f876ed15a00], txn-recover: Main recovery loop: starting at 1/23552
2018-11-07T01:30:41.059+0000 I STORAGE  [initandlisten] WiredTiger message [1541554241:59828][1:0x7f876ed15a00], txn-recover: Recovering log 1 through 2
2018-11-07T01:30:41.436+0000 I STORAGE  [initandlisten] WiredTiger message [1541554241:436051][1:0x7f876ed15a00], file:sizeStorer.wt, txn-recover: Recovering log 2 through 2
2018-11-07T01:30:41.486+0000 I STORAGE  [initandlisten] WiredTiger message [1541554241:486493][1:0x7f876ed15a00], file:sizeStorer.wt, txn-recover: Set global recovery timestamp: 0
2018-11-07T01:30:41.507+0000 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2018-11-07T01:30:41.536+0000 W STORAGE  [initandlisten] Detected configuration for non-active storage engine mmapv1 when current storage engine is wiredTiger
2018-11-07T01:30:41.536+0000 I CONTROL  [initandlisten] 
2018-11-07T01:30:41.537+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2018-11-07T01:30:41.537+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2018-11-07T01:30:41.537+0000 I CONTROL  [initandlisten] 
2018-11-07T01:30:41.538+0000 W CONTROL  [initandlisten] 
2018-11-07T01:30:41.538+0000 W CONTROL  [initandlisten] 
2018-11-07T01:30:41.538+0000 I CONTROL  [initandlisten] 
2018-11-07T01:30:41.557+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/usr/data/db/diagnostic.data'
2018-11-07T01:30:41.558+0000 I NETWORK  [initandlisten] waiting for connections on port 27017
2018-11-07T01:30:42.009+0000 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2018-11-07T01:30:53.699+0000 I NETWORK  [listener] connection accepted from 127.0.0.1:47378 #1 (1 connection now open)
2018-11-07T01:30:53.700+0000 I NETWORK  [conn1] received client metadata from 127.0.0.1:47378 conn1: { application: { name: "MongoDB Shell" }, driver: { name: "MongoDB Internal Client", version: "4.0.3" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2018-11-07T01:31:48.626+0000 I NETWORK  [conn1] end connection 127.0.0.1:47378 (0 connections now open)
